lib/huginn_acumen_order_agent/acumen_order_agent.rb

```diff
@@ -22,6 +22,7 @@ class AcumenOrderAgent < Agent
         * endpoint: The root URL for the Acumen API
         * site_code: The site code from Acumen
         * password: The Acumen API password
+        * output_mode  - not required ('clean' or 'merge', defaults to 'clean')
```

Comment:
@Jhalter5Stones It doesn't currently default to clean.

You have the `default_options` that prefill the field as `clean`, but you have no logic defaulting the field it it's not provided on the config.

Reply:
No this is fine, we are doing the exact same thing in the BC order agent and it works as expected if output_mode doesn't exist